packages/govern-console/src/containers/DAO/DaoActionPage.tsx

```diff
@@ -0,0 +1,203 @@
+import React, { useState } from 'react';
+import { DropDown, Button, GU, SearchInput, IconDown, Grid, GridItem, useLayout } from '@aragon/ui';
+import styled from 'styled-components';
+import { useHistory } from 'react-router-dom';
+
+import DaoActionCard from './components/DaoActionCard/DaoActionCard';
+
+type actionsType = {
+  __typename?: string;
+  id: string;
+  state: string;
+  createdAt: string;
+  payload: {
+    __typename?: string;
+    id: string;
+    executionTime: string;
+    title: string;
+  };
+}[];
+
+type props = {
+  fetchMore: () => Promise<void>;
+  actions: actionsType;
+  isMore: boolean;
+  identifier: string;
+};
+
+const Container = styled.div`
+  margin-top: ${3 * GU}px;
+`;
+
+const HeaderContainer = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  margin-bottom: ${3 * GU}px;
+`;
+
+const Title = styled.div`
+  font-weight: 600;
+  font-size: 28px;
+  line-height: 125%;
+`;
+
+const SearchContainer = styled.div`
+  display: flex;
+  align-items: center;
+  width: 100%;
+  margin-bottom: ${3 * GU}px;
+`;
+
+const CustomActionButton = styled(Button)`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  padding: 12px 24px;
+  width: 134px;
+  height: 44px;
+  border-radius: 12px;
+  box-shadow: none;
+`;
+
+const LoadMoreButton = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  padding: 12px 24px;
+  width: 154px;
+  height: 44px;
+  border-radius: 12px;
+  color: #7483ab;
+  background: #ffffff;
+  box-shadow: none;
+  cursor: pointer;
+  & > svg {
+    padding-left: 2px;
+  }
+`;
+
+const ActionListContainer = styled.div`
+  width: 100%;
+  margin-bottom: ${3 * GU}px;
+`;
+
+const NoAvailableActionContainer = styled.strong`
+  text-align: center;
+`;
+
+const DaoActionsPage: React.FC<props> = ({ fetchMore, actions, isMore, identifier }) => {
+  const history = useHistory();
+  const [selected, setSelected] = useState<number>(0);
+  const [value, setValue] = useState<string>('');
+
+  const FilterState = (data: actionsType[0]) => {
+    enum Filters { // Convern DropDown Number to Status
+      Executable = 1,
+      Scheduled = 2,
+      Challenged = 3,
+      Executed = 4,
+      Ruled_Negatively = 5,
+    }
+    switch (selected) {
+      case Filters.Executable:
+        return data.state === 'Executable';
+      case Filters.Scheduled:
+        return data.state === 'Scheduled';
+      case Filters.Challenged:
+        return data.state === 'Challenged';
+      case Filters.Executed:
+        return data.state === 'Executed';
+      case Filters.Ruled_Negatively:
+        return data.state === 'Ruled Negatively';
+      default:
+        return data;
+    }

```

Comment:
should return true for default case instead of `data`?
Also, may be the code can be simplified by not using the switch statement.  

```
   const actionStates = ['All Actions', 'Executable'...]
   if (selected > 0) {
       return actionStates[selected] === data.state
   }

the actionStates can be used in the Dropdown items as well, eliminating defining the action description in 2 places.
```